.buildkite/pipeline.yml

```diff
@@ -24,4 +24,5 @@ steps:
 
   - label: ':hammer: tests'
     commands:
-      - "make test"
+      - "make"
+      - "KERNELIMAGE=/var/lib/fc-ci/vmlinux.bin ROOTDRIVE=/dev/null make test"
```

Comment:
Any reason not to make `/dev/null` the default in main_test.go?

Reply:
That'd make too much sense. Haha, fixed.